storage/innobase/trx/trx0roll.cc

```diff
@@ -452,9 +452,7 @@ trx_rollback_to_savepoint_for_mysql_low(
 	trx_mark_sql_stat_end(trx);
 
 	trx->op_info = "";
-#ifdef WITH_WSREP
-	trx->lock.was_chosen_as_wsrep_victim = false;
-#endif
+
```

Comment:
why is this patch needed? It does not seem to relate to high level task of ensuring that trx object does not have was_chosen_as_wsrep_victim set when trx is pulled from cache

Reply:
My understanding is that BF aborted thread will return trx to pool. Reusing same trx later will us
trx_init where this flag is reset.
I have removed this part as it looks redundant to me afaik.